roles/service-dehydrated/tasks/main.yml

```diff
@@ -0,0 +1,127 @@
+---
+- name: Install dehydrated dependencies
+  apt: name={{ dehydrated_dependencies }}
+
+- name: Checkout dehydrated from github
+  git:
+    repo: "{{ dehydrated_repo_url }}"
+    update: "{{ dehydrated_update }}"
+    dest: "{{ dehydrated_install_root }}"
+    version: "{{ dehydrated_version }}"
+
+- name: Checkout pdns_api.sh from github
+  git:
+    repo: "{{ pdns_api_repo_url }}"
+    update: "{{ pdns_api_update }}"
+    dest: "{{ dehydrated_install_root }}/pdns_api"
+    version: "{{ pdns_api_version }}"
+
+- name: Create /etc/dehydrated
+  file: dest=/etc/dehydrated state=directory owner=root group=root mode=0700
+
+- name: Generate dehydrated config
+  template:
+    dest: /etc/dehydrated/config
+    src: config.j2
+    owner: root
+    group: root
+    mode: 0600
+
+- name: Generate dehydrated domains.txt
+  copy:
+    dest: /etc/dehydrated/domains.txt
+    content: "{{ dehydrated_domains }}"
+    owner: root
+    group: root
+    mode: 0600
+  notify: run dehydrated
+
+- import_tasks: domain_config.yml
+
+- name: Generate deploycert.sh
+  template:
+    src: deploycert.j2
+    dest: /etc/dehydrated/deploycert.sh
+    owner: root
+    group: root
+    mode: "0700"
+  when: dehydrated_deploycert is defined
+
+- name: Remove deploycert.sh
+  file: dest=/etc/dehydrated/deploycert.sh state=absent
+  when: dehydrated_deploycert is not defined
+
+- name: Install cronjob
+  cron:
+    name: dehydrated-renew
+    minute: "{{ 59|random(seed=inventory_hostname) }}"
+    hour: "{{ 4|random(seed=inventory_hostname) }}"
+    user: root
+    job: "{{ dehydrated_install_root }}/dehydrated -c"
```

Comment:
stdout sollten wir nach /dev/null oder in irgendeine Datei umleiten damit wir nicht jeden Tag eine Email bekomme wenn der dehydrated cronjob läuft. War auf Zuckerwatte auch so eingerichtet.